src/components/SiteTag/SiteTag.tsx

```diff
@@ -6,36 +6,33 @@ import styles from './styles';
 
 export default function SiteTag(props) {
   const { value, onPress } = props;
+
   return (
     <View style={styles.container}>
       <TouchableHighlight
         underlayColor="#aaa"
         onPress={onPress}
       >
-        <View
-          // pointerEvents="none" is a workaround to get Touchable and Input to work together:
```

Comment:
I removed this comment because I don't think it's a "workaround" so much as the way you do things in React Native in order to disambiguate which element in the tree hierarchy should handle pointer or touch events.